.github/workflows/CI.yml

```diff
@@ -48,6 +48,11 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@v1
+      - name: Set and Retrieve Github ENV variables
+        shell: bash
+        run: |
+          export JULIA_NUM_THREADS=2
```

Comment:
This line doesn't do anything because only the GITHUB_ENV is persistent across steps